### PR TITLE
sample

### PR DESCRIPTION
**Pull request by charry1729**
**Changelog**
- Add
- Fix
- Update

 

**Reviewed by**
@developer_github_id

 

**Linked issue**
#issue_number

### CODE REVIEW

**Comment by sownak** [comment]
Please check https://github.com/hyperledger/bevel/blob/main/CONTRIBUTING.md

File: platforms/hyperledger-besu/charts/node_besu/templates/deployment.yaml [hunk 1]
```diff
@@ -221,7 +221,7 @@ spec:
             --privacy-enabled=true \
             --privacy-public-key-file=/secrets/tm_key.pub \
             --Xdns-enabled=true \
-            --Xdns-update-enabled=true \
+            --Xdns-update-enabled=true \z
```
Comment (jagpreetsinghsasan) [inline platforms/hyperledger-besu/charts/node_besu/templates/deployment.yaml:224]: What is this `z` for?

File: platforms/shared/charts/besu-connector/templates/configmap.yaml [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    {  
+    "packageName" : {{ .Values.plugins.packageName | quote }},
+    "type" : {{ .Values.plugins.type | quote }},
+    "action" : {{ .Values.plugins.action | quote }},
+    "options": {
+      "rpcApiHttpHost" : {{ .Values.plugins.rpcApiHttpHost | quote }},
+      "rpcApiWsHost" : {{ .Values.plugins.rpcApiWsHost | quote }},
+      "instanceId" : {{ .Values.plugins.instanceId | quote }} 
+      }
+    }
```
Comment (jagpreetsinghsasan) [inline platforms/shared/charts/besu-connector/templates/configmap.yaml:30]: Please add a new line here

File: platforms/shared/charts/besu-connector/templates/deployment.yaml [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+  labels:
+    app.kubernetes.io/name: {{ .Values.service.name }}
+    helm.sh/chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}
+    app.kubernetes.io/managed-by: {{ .Release.Service }}
+    app.kubernetes.io/instance: {{ .Release.Name }} 
+    {{- include "labels.custom" . | nindent 2 }}
+  name: {{ .Values.service.name }}
+
+spec:
+  # serviceName: {{ .Values.service.name }}
```
Comment (jagpreetsinghsasan) [inline platforms/shared/charts/besu-connector/templates/deployment.yaml:15]: Please remove unwanted commented code everywhere

File: platforms/shared/charts/besu-connector/templates/service.yaml [hunk 4]
```diff
@@ -0,0 +1,46 @@
+apiVersion: v1
+kind: Service
+metadata:
+  namespace: {{ .Values.metadata.namespace }}
+  name: {{ .Values.service.name | quote }}
+  annotations:
+    app: consortiumchain
+  # version: '1'
```
Comment (jagpreetsinghsasan) [inline platforms/shared/charts/besu-connector/templates/service.yaml:8]: Connectors will generally stay outside the ledger environment, thus having ambassador mappings is must

File: platforms/shared/charts/besu-connector/values.yaml [hunk 5]
```diff
@@ -0,0 +1,57 @@
+
+metadata:
```
Comment (jagpreetsinghsasan) [inline platforms/shared/charts/besu-connector/values.yaml:2]: Please add in-line comments for each `key:value` pair

File: platforms/shared/charts/fabric-connector/templates/configmap.yaml [hunk 6]
```diff
@@ -0,0 +1,54 @@
...
+        "CORE_PEER_TLS_ROOTCERT_FILE": {{ .Values.plugins.corePeerTlsRootcertFile | quote }} ,
+        "ORDERER_TLS_ROOTCERT_FILE": {{ .Values.plugins.ordererTlsRootcertFile | quote }} 
+      },
+    "discoveryOptions": {
+      "enabled": {{ .Values.plugins.enabled  | quote }} ,
+      "asLocalhost": {{ .Values.plugins.asLocalhost  | quote  }} 
+      }
+     }
+    }
+
```
Comment (jagpreetsinghsasan) [inline platforms/shared/charts/fabric-connector/templates/configmap.yaml:41]: Please do a cleanup of unwanted commented code or newlines

File: platforms/shared/charts/fabric-connector/templates/configmap.yaml [hunk 7]
```diff
@@ -0,0 +1,54 @@
...
+  labels:
+    app.kubernetes.io/name: plugins-{{ .Values.service.name }}
+    helm.sh/chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}
+    app.kubernetes.io/managed-by: {{ .Release.Service }}
+    app.kubernetes.io/instance: {{ .Release.Name }}
+    {{- include "labels.custom" . | nindent 2 }}
+data:
+  PLUGINS: |
+    {
+    "packageName": {{ .Values.plugins.packageName | quote }},
```
Comment (jagpreetsinghsasan) [inline platforms/shared/charts/fabric-connector/templates/configmap.yaml:22]: Please have proper tabs and spacings for this json config

**Review by jagpreetsinghsasan — CHANGES_REQUESTED** [review]
There are a lot of more files to be reviewed. Please cleanup the code, add incode comments, remove unwanted commented code as soon as possible, we have very little time left for this.

Also, `let this PR be in draft phase until said so` because there are hundreds of commits, improper contributing guidelines; all of these should be fixed in the draft PR itself before creating the final `good-to-go` PR

**Comment by sownak** [comment]
This is not a valid request and the changes will be incorporated in a separate PR that I have worked upon.

**Pull request closed without merging**